helm/spark-yotpo/spark-values.yaml

```diff
@@ -0,0 +1,32 @@
+version: 0.2
```

Comment:
@giladw im missing all the default values from spark-defaults.conf

Reply:
those values should be inside the spark conf?
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/user-guide.md#specifying-spark-configuration